### PR TITLE
JBPM-8555 : Stunner - Align and distribution not working properly

### PR DESCRIPTION
**Pull request by inodeman**
Hi @hasys can you review this PR? It solves the alignment issue horizontally and vertically, basically it only does the getBoundingBox calculation without the text in the shapes. I am fairly new to this, and I tested it well to the best of my knowledge but want to make sure that calculation does not affect anything else. Let me know

### CODE REVIEW

**Comment by inodeman** [comment]
Jenkins execute full downstream build

**Comment by tiagodolphine** [comment]
Jenkins execute full downstream build

**Comment by hasys** [comment]
Jenkins retest please

**Comment by hasys** [comment]
### Check list
- [x] Affected version is set property
- [x] Doc team notified?
- [x] Is realization aligned to jira description?
- [x] Both Jenkins jobs are green

### Code review
- [x] Code coverage
  - missing some tests for internal logging classes, not critical
- [x] License headers
- [x] Static analyser gates
- [x] Code Style

### Acceptance Criteria
> * Model Start->Task->End, Move Task down or up, try to align Start->Task->End back
> * Same as first but inside of the Sub-Process and Swimlanes.
> * Same as first but with other nodes on the canvas

- [x] Tested with full downstream build
  - [x] Firefox
  - [x] Chrome

### Found issues
>  **Blocker**, **Critical** and **Major** bugs and **Blocker** and **Critical** enhancements should be fixed before merge.
* None

**Comment by hasys** [comment]
Hi @manstis, @ederign,

this PR is ready to be merged.

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-lienzo/src/main/java/org/kie/workbench/common/stunner/client/lienzo/shape/view/wires/ext/WiresLayoutContainerNoTextBoundingBox.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+/**
+ * The purpose of this class is to allow Items to align without taking account Text.
+ * JBPM-8555
+ */
+public class WiresLayoutContainerNoTextBoundingBox extends WiresLayoutContainer {
+
+    public WiresLayoutContainerNoTextBoundingBox() {
+
+        super(new Group() {
```
Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-lienzo/src/main/java/org/kie/workbench/common/stunner/client/lienzo/shape/view/wires/ext/WiresLayoutContainerNoTextBoundingBox.java]: Just for the records - It maybe results more usefull to create a new subtype of `Group`, something like "FilteredBoundingBoxGroup", which overrides the `getBoundingBox` method where the conditional sentence is being provided in some `Predicate` field (which can be set via constructor).  Then it results natural to create a factory class that creates a custom Group which filter the `Text` nodes from the BB... and it results easier to re-use it for other conditions, just by settings different `Predicate` arguments.
Anyway just a minor thing, to keep in mind for next commits, if you think it makes sense... :) Good job! :+1:

**Pull request merged**